src/pages/Search/SearchTypeMenu.tsx

```diff
@@ -254,6 +254,8 @@ function SearchTypeMenu({queryJSON, searchName}: SearchTypeMenuProps) {
                             wrapperStyle={styles.sectionMenuItem}
                             focused={index === activeItemIndex}
                             onPress={onPress}
+                            shouldUseAutoWidth
+                            style={styles.pv4}
```

Comment:
Added a new prop to handle the default background width and height for the MenuItem Icon. The default is currently set to 40. For the nav bar items, we don't need to pass any default height or width for the Icon container, so I used this prop to manage the height and width of the Icon container on the Settings, Workspace Details, and Search Tab pages only

Reply:
> Will also need to pass iconType='icon'

Sorry, I didn't quite understand. Where are you saying we need to pass `iconType='icon'`? On `MenuItem`, we’re already setting the default as `icon` here.

https://github.com/Expensify/App/blob/f251dbcdfdafc492d9a9908cbf40e4630b7b6d9d/src/components/MenuItem.tsx#L359

Reply:
> Instead of adding this new prop, we should fix the MenuItem to use the correct size. It should use iconSizeNormal instead of calling getAvatarWidthStyle since the passed icon is not an avatar.

 Could you please elaborate? Are you suggesting we remove `getAvatarWidthStyle` altogether and instead use `{width: iconSizeNormal}`? If that's the case, I want to point out that it will also affect the margin of the FAB menu context items
<img width="375" alt="Screenshot 2024-10-23 at 15 29 32" src="https://github.com/user-attachments/assets/bb196bfd-aceb-43a2-9303-7d300a1c7a12">

Reply:
We don't want to do anything that changes the option rows/global create item styles

Reply:
The problem is that MenuItem uses the sizes of avatars for icons which is wrong. But yeah let's not worry about this here. Can you just rename the `shouldUseAutoWidth` prop to `shouldIconUseAutoWidthStyle`.

Reply:
Yes sure renaming it now